parent-pom/pom.xml

```diff
@@ -67,7 +67,7 @@
             <dependency>
                 <groupId>org.hobbit</groupId>
                 <artifactId>core</artifactId>
-                <version>1.0.10</version>
+                <version>1.0.10-SNAPSHOT</version>

```

Comment:
Using a `SNAPSHOT` version is not a good idea if a stable version with the same version number has been used before. It is like "downgrading" the version.